apisix/plugins/prometheus/exporter.lua

```diff
@@ -351,6 +356,42 @@ local function etcd_modify_index()
 
 end
 
+local key = {}
+local combine_param = {}
+
+local function shared_dict_status()
+    local header_of_shared_dict = core.request.header(ngx.ctx, "shared_dict")
+    if not header_of_shared_dict then
+        return
+    end
+
+    local set_shared_dict_param = function (shared_dict_name)
+        if type(shared_dict_name) ~= "string" then
+            return
+        end
+        local share_dict = ngx.shared[shared_dict_name]
+        if share_dict then
+            combine_param[1] = shared_dict_name
+            combine_param[2] = "capacity"
+            key[1] = concat_tab(combine_param, "_")
+            metrics.shared_dict:set(share_dict:capacity(), key)
+
+            combine_param[2] = "free_space"
+            key[1] = concat_tab(combine_param, "_")
+            metrics.shared_dict:set(share_dict:free_space(), key)
+        end
+    end
```

Comment:
can we move this function between `local combine_param = {}` and `local function shared_dict_status()`?

Reply:
> can we move this function between `local combine_param = {}` and `local function shared_dict_status()`?

can you tell me why do this?

Reply:
I thought it over and it didn't seem necessary. ignore this.